### ISSUE TITLE
On 1.7.10, server crashes while trying to open the alternative GUI for a diamond backpack with depth upgrade

### ISSUE THREAD

**Issue by depressed-pho** [gr8pefish/IronBackpacks#191]
Stack trace:
```
java.lang.ArrayIndexOutOfBoundsException: 4
        at main.ironbackpacks.items.backpacks.ItemBackpack.onItemUseFirst(ItemBa
ckpack.java:96)
        at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemI
nWorldManager.java:360)
...
```

https://github.com/gr8pefish/IronBackpacks/blob/7deb7518c5781e25a791ae305f5b276af815c929/src/main/java/main/ironbackpacks/items/backpacks/ItemBackpack.java#L96

Shouldn't it be ```itemBackpack.getGuiId()```?

**Comment by gr8pefish** [comment]
I might fix this one if I have my 1.7 development environment up, but 1.7 is officially unsupported now. Regardless, it won't be until I get home, which wont be until January.

**Comment by TehNut** [comment]
Sorry, we're not accepting issues for 1.7.10, anymore!
